core/lib/Cargo.toml

```diff
@@ -40,6 +40,7 @@ base64 = "0.10"
 pear = "0.1"
 atty = "0.2"
 async-std = "0.99.4"
+either = "1.5.2"
```

Comment:
Is there a reason to depend on `1.5.2` vs. just `1.0`?

Reply:
It's just what was added when I used `cargo add`. There shouldn't be any particular reason to require the higher version.